packages/ui-extensions/src/surfaces/admin/components/ColorField/examples/basic-usage-required.html

```diff
@@ -0,0 +1,8 @@
+<s-stack gap="base">
+  <s-color-field
+    label="Brand color"
+    value="#FF0000"
```

Comment:
Value specified twice here, either remove or replace with defaultValue
```suggestion
```

or 
```suggestion
defaultValue="#0000FF"
```